contracts/adapters/ReinvestmentExtensionV1.sol

```diff
@@ -144,16 +169,38 @@ contract ReinvestmentExtensionV1 is BaseExtension {
     }
 
     /**
-     * APPROVED_CALLER ONLY: Updates execution settings for a reward token
-     * 
-     * @param _rewardToken    Address of reward token
-     * @param _settings       New execution settings
+     * OPERATOR ONLY: Updates exchange settings for a reward token
      */
-    function updateExecutionSettings(
+    function updateExchangeSettings(
         address _rewardToken,
-        ExecutionSettings memory _settings
-    ) external onlyAllowedCaller(msg.sender) {
-        settings[_rewardToken] = _settings;
+        ExchangeSettings memory _settings
+    ) external onlyOperator {
+        require(_rewardToken != address(0), "Invalid reward token");
+        exchangeSettings[_rewardToken] = _settings;
+    }
+
+    /**
+     * OPERATOR ONLY: Adds an approved wrap pair
+     * 
+     * @param _underlyingToken    Address of underlying token
+     * @param _wrappedToken       Address of wrapped token
+     */
+    function addWrapPair(address _underlyingToken, address _wrappedToken) external onlyOperator {
+        require(_underlyingToken != address(0) && _wrappedToken != address(0), "Invalid token address");
+        require(!approvedWrapPairs[_underlyingToken][_wrappedToken], "Pair already exists");
+        approvedWrapPairs[_underlyingToken][_wrappedToken] = true;
+    }
+
+    /**
+     * OPERATOR ONLY: Removes an approved wrap pair
+     * 
+     * @param _underlyingToken    Address of underlying token
+     * @param _wrappedToken       Address of wrapped token
+     */
+    function removeWrapPair(address _underlyingToken, address _wrappedToken) external onlyOperator {
+        require(_underlyingToken != address(0) && _wrappedToken != address(0), "Invalid token address");
+        require(approvedWrapPairs[_underlyingToken][_wrappedToken], "Pair does not exist");
+        approvedWrapPairs[_underlyingToken][_wrappedToken] = false;
```

Comment:
minor: I think deleting is more gas efficient than setting to false?

Reply:
added here [`716b1ff`](https://github.com/IndexCoop/index-coop-smart-contracts/pull/201/commits/716b1ffe7b5eb6efb9c9f237d451cf3c11441c8d)